src/main/java/AddCommand.java

```diff
@@ -0,0 +1,60 @@
+package duke.command;
+
+import duke.TaskList;
+import duke.Ui;
+import duke.Storage;
+import duke.task.Task;
+
+/**
+ * Represents a Command that adds a Task to the TaskList.
+ * An AddCommand consists of a Task.
+ *
+ */
+
```

Comment:
The coding standard used for this module suggests leaving no blank line between the documentation block and the method/class. Perhaps it might save some space too?